create_infotexts.py

```diff
@@ -387,7 +368,23 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
                 self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
-            place_as_wikitext += place_string
+            place_matches = []
+            for place in places_mapping:
+                if place.lower() in desc_string.lower():
+                    if places_mapping[place]["wikidata"]:
+                        place_matches.append("{{{{city|1={wikidata}}}}}".format(
+                            wikidata=places_mapping[place]["wikidata"]))
+                    else:
+                        place_matches.append(place)
+
+                    # Don't forget to add the commons categories if present.
+                    if places_mapping[place].get('commonscat'):
+                        self.content_cats.append(places_mapping[place]["commonscat"])
+
+            if len(place_matches) == 0:
+                self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
```

Comment:
With the current implementation place_string will always be None if len(place_matches) == 0, so no use adding that.

Not done.